### DISCUSSION TITLE
Connect io.sentry.logback to wildfly 24

### DISCUSSION THREAD

**Discussion by huadon** [getsentry/sentry-java#2129] [General]
Is it possible to connect sentry to wildfly as part of custom-handler for logging?

tried adding io.sentry and io.sentry.logback as module but not working.

**Comment by maciejwalkowiak** [comment]
I've written some time ago a guide for setting up Sentry with JBoss (https://gist.github.com/maciejwalkowiak/02fa1eb0089ebc6e0a4687458a204779). Not sure how things are different in Wildfly 24 but perhaps it will be helpful for you.

**Reply by huadon** [reply]
Thank you I will try it. Also I guess I can use latest version of all libs

**Reply by maciejwalkowiak** [reply]
Yes, note that starting from Sentry SDK 6.x, GSON is not used so you can skip it. Since JBoss uses JUL for logging, in this sample config I've used Sentry JUL integration. Not sure how is it going to work with Logback but would definitely appreciate if you can share your learnings.

**Reply by huadon** [reply]
try to follow instruction with latest 6.1.3 sentry and sentry-jul, but always got following error

 Failed to link io/sentry/jul/SentryHandler (Module "io.sentry" from local module loader @618c5d94 (finder: local module finder @5b40ceb (roots: C:\workfolder\wildfly-24.0.0.Final\wildfly-24.0.0.Final\modules,C:\workfolder\wildfly-24.0.0.Final\wildfly-24.0.0.Final\modules\system\layers\base))): java/util/logging/Handler

**Reply by huadon** [reply]
ok, it worked after i added dependency in module.xml to
<module name="javax.api" />
    <module name="javax.servlet.api" />
    <module name="org.slf4j"/>

**Reply by maciejwalkowiak** [reply]
Just checked and it does work for me
